docs/getting-started.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Getting Started
+weight: 80
+---
+
+## Introduction
+
```

Comment:
@Souvikns I removed that intro part from that other PR and added it here like nextjs docs follows